annotations/templates/annotations/header.html

```diff
@@ -88,9 +88,7 @@
                         <ul class="dropdown-menu">
                             <li><a href="{% url "concepts" %}">Concepts</a></li>
                             <li><a href="{% url "relations" %}">Annotations</a></li>
-                            {% if user.is_staff %}
-                                <li><a href="{% url "list_relationtemplate" %}">Templates</a></li>
-                            {% endif %}
+                            {% if user.is_staff or user.is_vogon_admin %}<li><a href="{% url "list_relationtemplate" %}">Templates</a></li>{% endif %}
```

Comment:
probably best to be consistent with your quoting syntax. Various {% %} blocks utilize double quotes inside and outside whereas some utilize double quotes outside and single quotes inside (I think single quotes inside is generally the popular choice)